### PR TITLE
Create applicant button

### PR DESCRIPTION
**Pull request by andsemenov**
Acceptance Criteria:
- button reads "Log in as Applicant"
- button is placed below page header/title
- button is a link to "/applicant_login"

### CODE REVIEW

File: client/src/components/Home.js [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
+    history.push("/applicant_login");
+  }
+
   return (
     <div>
       <header className="home-header">
         <h1>Diversi-Hire</h1>
       </header>
+      <Button primary onClick={loginClick}>
+        I am an applicant
```
Comment (gregdyke) [inline client/src/components/Home.js]: If we're strictly following the acceptance criteria it should read "Log in as applicant", but as I arbitrarily chose that today, I don't think it matters much

**Review by gregdyke — COMMENTED** [review]
(I don't know enough about react to know whether "useHistory" is the "correct" way of doing this, so I'll leave the review to the TAs)

File: client/src/components/Home.js (hunk 1 above)
Comment (andsemenov) [reply client/src/components/Home.js]: I've fixed. Could you review, please? Thanks.

File: client/src/components/Home.js [hunk 2]
```diff
@@ -1,12 +1,18 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import { Button } from "semantic-ui-react";
 import "../styles/Home.css";
+import "semantic-ui-css/semantic.min.css";
```
Comment (llh1) [inline client/src/components/Home.js]: Can this be imported in the index.js file instead? It just needs to be imported once in the project and Sharlu's PR6 will also include it in the index.js. So we can make sure we import it in the same place for everyone.

Comment (andsemenov) [reply client/src/components/Home.js]: Ok, that's right. I think index.js is more proper place for this. Thanks.

**Review by llh1 — APPROVED** [review]

**Pull request merged**